.github/actions/devcontainer_run_command/action.yml

```diff
@@ -115,6 +115,10 @@ inputs:
     description: "A value indicating if resources with state will be protected with locks."
     required: false
     default: "true"
+  TF_VAR_enable_airlock_malware_scanning:
+    description: "If False, Airlock requests will skip the malware scanning stage."
+    required: true
```

Comment:
1. are you sure you need this with required? 
2. you don't have enough changes in this PR to support this definition - I believe some runs are likely to fail as this isn't in all places call on this custom action